src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -121,7 +121,7 @@ protected void handlePut(RestChannel channel, final RestRequest request, final C
             final SecurityDynamicConfiguration<?> rolesConfiguration = load(CType.ROLES, false);
             for (final String role: opendistroSecurityRoles) {
 
-                if (!rolesConfiguration.exists(role) || isHidden(rolesConfiguration, role)) {
+                if (!rolesConfiguration.exists(role) || rolesConfiguration.isHidden(role)) {
```

Comment:
It will be good to understand the desired behavior here. If super admin is able to see a role, should she be able to assign that role to a user? @debjanibnrj